README.md

```diff
@@ -1,26 +1,4 @@
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-Enables AWS Config and adds managed config rules with good defaults.
```

Comment:
There's a change in functionality in `terraform-docs` were the tool no longer generates the usage docs from main.tf. To fix this could you copy over the docs from https://github.com/trussworks/terraform-aws-config/blob/master/main.tf#L2 and add it to the README?

Reply:
@dynamike thanks for your time and review here, commit in place based on your change rec.   

- 98e3b91 - adding usage doc to README.md since terraform-docs does not generate it any more.

CC: @diego-ojeda-binbash